g/go-oci8/go-oci8.sh

```diff
@@ -0,0 +1,73 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: go-oci8
+# Version	: v0.0.7
+# Source repo	: https://github.com/mattn/go-oci8
+# Tested on	: UBI 8.5
+# Language      : GO
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Atharv Phadnis <Atharv.Phadnis@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=github.com/mattn/go-oci8
+PACKAGE_VERSION=${1:-v0.0.7}
+
+set -e
+
+yum install -y git gcc-c++ golang libaio libnsl2 unzip wget
+ln -s /usr/lib64/libnsl.so.2 /usr/lib64/libnsl.so.1
+
+mkdir -p /opt/oracle && cd /opt/oracle
+wget https://download.oracle.com/otn_software/linux/instantclient/193/instantclient-basic-linux.leppc64.c64-19.3.0.0.0dbru.zip
+unzip instantclient-basic-linux.leppc64.c64-19.3.0.0.0dbru.zip
+rm -f instantclient-basic-linux.leppc64.c64-19.3.0.0.0dbru.zip
+wget https://download.oracle.com/otn_software/linux/instantclient/193/instantclient-sdk-linux.leppc64.c64-19.3.0.0.0dbru.zip
+unzip instantclient-sdk-linux.leppc64.c64-19.3.0.0.0dbru.zip
+rm -f instantclient-sdk-linux.leppc64.c64-19.3.0.0.0dbru.zip
+echo /opt/oracle/instantclient_19_3 > /etc/ld.so.conf.d/oracle-instantclient.conf
+ldconfig
```

Comment:
Can we add a comment describing why this is required? Whether it is build/test dependency or required for something else.

Reply:
Made the requested changes